tests/test_community_scripts_proxmoxve.py

```
@@ -86,7 +102,7 @@ def load_engine_module() -> tuple[types.ModuleType, DummyLogger]:
     result_types_module.EngineResults = DummyEngineResults
 
     network_module = types.ModuleType("searx.network")
-    network_module.get = lambda url, timeout: FakeHTTPResponse([])
+    network_module.get = lambda url, timeout: FakeHTTPResponse(_pocketbase_response([]))
```

Ruff (0.15.9)

[warning] Unused lambda argument: `url`

(ARG005)